docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+. If you have not already installed the App Studio GitHub application, select Install GitHub application on the “Manage build pipelines” popup page, select *Install Github application*. 
+
++
+NOTE: If you want to restrict the GitHub application’s access to certain repositories only, use the Only select repositories option in GitHub during the installation.
+
+. Returning to the “Manage build pipelines” page in {ProductName}, select *Send pull request*.
+. Select *Merge in GitHub*.
+. In GitHub, merge the pull request from the “red-hat-trusted-app-pipeline” bot.
+. Let {ProductName} complete PipelineRun for the upgraded build pipeline.
+
+.Verification
+
+. Go to *Activity > Pipeline* runs.
+
+. Select the most recent *PipelineRun*.
+
+. View the +++ <span class="popup"><code>fbc-label-check</code><span class="popuptext">Ensures that FBC image have all necessary labels.</span> </span>, <span class="popup"><code>fbc-validate</code><span class="popuptext">Checks the existence and functionality of binaries, and runs validation on configs within the image.</span> </span>, and the <code>fbc-related-image-check</code> task within the build pipeline. Ensure that the build process is executing these tasks and not indicating a “skip checks” status. If these tasks show a green checkmark, it indicates that the build process has validated the fbc components against an existing link:https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels_package[Enterprise Contract] policy. However, if they do not show a green checkmark, it indicates some potential issues in FBC components. +++
+
+NOTE: We now have the image, next step is to release the image for which we need to set up the workspace.
+
+== Setting up workspaces to release an FBC application
+
+Two teams work together to release an application:
+
+* *Development team*- The team that develops and supports the application in a pre-production environment.
+* *Managed environment team* - The team that supports the production instance of that application.
+
+To release an application to a managed environment, the development team creates a `ReleasePlan` object in the developer workspace. The `ReleasePlan` object includes a reference to the application that the development team wants to release, along with the namespace and workspace where the application is supposed to release.
+
+=== Creating a ReleasePlan object
+
+.Prerequisites
+
+* You have an existing Development and Managed workspace.
+* You have installed `oc`.
+* You have completed the steps listed in the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[Getting started in the CLI] page. 
+
+.Procedures
+
+* In the development workspace, create a `release_plan.yaml` object by running: `oc apply -f releaseplan.yaml -n dev`. It adds the required configuration to your workspace that is needed to execute a release pipeline in a managed workspace.
```

Comment:
Can the command become a block so that it's not in-line? E.g.:
```suggestion
* In the development workspace, create a `release_plan.yaml` file by running the following command:
+
[source,terminal]
----
$ oc apply -f releaseplan.yaml -n dev
----
+
This command adds the required configuration to your workspace that is needed to execute a release pipeline in a managed workspace.
```